src/app/views.py

```diff
@@ -103,6 +105,24 @@ def generateLicenseXml(licenseOsi, licenseIdentifier, licenseName, licenseSource
     xmlString = ET.tostring(root, encoding='utf8', method='xml')
     return xmlString
 
+def createIssue(licenseName, licenseIdentifier, licenseSourceUrls, licenseOsi):
+    """ View for creating an GitbHub issue
+    when submitting a new license request
+    """
+    myToken = ''
+    body = '**1.** License Name: ' + licenseName + '\n**2.** Short identifier: ' + licenseIdentifier + '\n**3.** URL: '
+    print 'longitud: ' + str(licenseSourceUrls)
+    for url in licenseSourceUrls:
+        body += url
+        body += '\n'
+    body += '**4.** OSI Approval: ' + licenseOsi
+    title = 'New license request: ' + licenseIdentifier + ' [SPDX-Online-Tools]'
+    payload = {'title' : title, 'body': body, 'labels': ['new license/exception request']}
+    headers = {'Authorization': 'token ' + myToken}
+    url = 'https://api.github.com/repos/spdx/license-list-XML/issues'
+    r = post(url, data=dumps(payload), headers=headers)
+    print r.status_code
+
```

Comment:
We should add some error handling here - if the status is not a success, open a dialog "Please note that there was a problem opening the issue for the SPDX legal team.  Please email spdx-legal@lists.spdx.org with SPDX ID for the license you are submitting"

Reply:
Ok sure, I am going to be working on this.

Reply:
Now a warning message is displayed when the issue cannot be created. However, even when an issue could not be created, it could be stored on the submitted requests database. What do we do with requests whose issue cannot be created? Do we still save them or not? @goneall @rtgdk

Reply:
I would still save the info in the DB - the legal team could follow-up and create an issue manually using the information from the DB.  We could enhance this in the future to email the legal team that an error occurred, but I think the warning message is fine for now.